floris/simulation/base.py

```diff
@@ -37,13 +43,16 @@ class State(Enum):
     USED = 2
 
 
-class BaseClass(LoggerBase, FromDictMixin):
+@define
+class BaseClass(FromDictMixin):
     """
     BaseClass object class. This class does the logging and MixIn class inheritance.
     """
 
-    state = State.UNINITIALIZED
-
+    # Initialize `state` and ensure it is treated as an attribute rather than a constant parameter.
+    # See https://www.attrs.org/en/stable/api-attr.html#attr.ib
+    state = field(init=False, default=State.UNINITIALIZED)
+    _logging_manager: LoggingManager = field(init=False, default=LoggingManager())
```

Comment:
Yea the other form was overkill, in retrospect. Since this change, LoggingManager doesn't need to be an attrs class, so I removed the import you caught above.

Reply:
It's clear now that the previous method was overkill. Since LoggingManager isn't used as a super-class within `floris.simulation`, it shouldn't be an attrs class so I removed the import you pointed out above.